common/audio_desktop.go

```diff
@@ -0,0 +1,123 @@
+//+build !windows,!netgo,!android
```

Comment:
Hopefully, will be a little tricky though because of openal but I have an idea in mind.